lib/v2/bilibili/maintainer.js

```diff
@@ -42,4 +42,5 @@ module.exports = {
     '/vsearch/:kw/:order?/:disableEmbed?/:tid?': ['Symty'],
     '/watchlater/:uid/:disableEmbed?': ['JimenezLi'],
     '/weekly/:disableEmbed?': ['ttttmr'],
+    '/platform/:area?/:p_type?/:uid?': ['nightmare-mio'],
```

Comment:
Please insert new entries in [alphabetical order](https://docs.rsshub.app/joinus/advanced/script-standard#general-guidelines).